specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ContainerApps_ManagedBy_CreateOrUpdate.json

```diff
@@ -0,0 +1,204 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "containerAppName": "testcontainerAppManagedBy",
+    "api-version": "2022-11-01-preview",
+    "containerAppEnvelope": {
+      "location": "East US",
+      "managedBy": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.AppPlatform/Spring/springapp",
```

Comment:
This property looks good. 

One note is that the value of this property might be Spring *"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AppPlatform/Spring/{serviceName}"* or Spring App *"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AppPlatform/Spring/{serviceName}**/apps/{appName}**"*, and will be finilized after we settle down the Logs/Metrics integration with @howang-ms .